### DISCUSSION TITLE
Special Characters in URL

### DISCUSSION THREAD

**Discussion by 51Tx** [apache/kyuubi#6513] [General]
Setting URL parameters in the URL of dbeaver is similar to that of 
```
jdbc:kyuubi://host:port#xxx.url=jdbc:mysql://host:port/database
```
will have an Illegal character in fragment issue, and/will not be used after being replaced with `%2f`

**Comment by pan3793** [comment]
could you please try this way?
<img width="812" alt="image" src="https://github.com/apache/kyuubi/assets/26535726/00293a47-25ec-4a76-b390-a28306f155f5">

**Reply by 51Tx** [reply]
Thank you for your response
I mainly use this parameter to do some other logic in the code. Do I support this type of parameter passing in URLs?
![image](https://github.com/apache/kyuubi/assets/34480262/0f2d1c57-dcb3-43e4-b9a5-438906cfd8ca)
